config/environments/test.rb

```diff
@@ -19,8 +19,6 @@
   # loading is working properly before deploying your code.
   config.eager_load = ENV['CI'].present?
 
-  config.assets_digest = false
```

Comment:
I believe this is no longer relevant/needed based on what the propshaft/webpack interaction is.